RawWaterManagement/HydraulicSimulationScenario/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,144 @@
+{
+  "id": "urn:ngsi-ld:dataModel:id:GZAR:07210621",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1970-05-23T01:18:44Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2010-12-22T14:03:23Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Water remember theory vote fish. Speak hotel think wish. Nature find speech year may."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Day despite article rich agent deal mouth. Opportunity star set. Wish do generation able us party sell. Reveal approach building technology return movement positive."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "So product organization last."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Interview other plant pattern close policy focus. Well hotel goal ball suddenly third goal."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Heart may put room. Dog product hot reflect expect. Write modern spring industry on summer."
+  },
+  "owner": {
```

Comment:
1. Relationships should not be modelled as properties, otherwise you completely lost the interest of NGSI-LD which is based on graph. 
2. Arrays in NGSI-LD are stored as strings so you can not use the API to query indiviudual alements in a array. 
3. Relationships can not be provided within arrays.

When you need to have several relationships with the same name, then you have to use the datasetId 

Example for a relationship where one entity would "monitor" 2 different rooms. It thus have 2 relationshipsnames "monitored" directed toward room01 and room02. The datasetId allows to differentiate the 2 relationships.
"monitors":[
	{
	   "type":"Relationship",
	   "object":"urn:ngsi-ld:Room:01",
	   "datasetId":"urn:ngsi-ld:dataSet:01"},
	},
	"type":"Relationship",
	"object":"urn:ngsi-ld:Room:02" ,
	"datasetId":"urn:ngsi-ld:dataSet:02"}
	]